core/src/main/java/com/linecorp/armeria/server/ServerBuilder.java

```diff
@@ -1124,10 +1125,35 @@ public ServerBuilder tlsAllowUnsafeCiphers(boolean tlsAllowUnsafeCiphers) {
         return this;
     }
 
+    /**
+     * Returns a {@link ContextPathServicesBuilder} which binds {@link HttpService}s under the
+     * specified context paths.
+     *
+     * @see ContextPathServicesBuilder
+     */
+    @UnstableApi
+    public ContextPathServicesBuilder<ServerBuilder> contextPath(String... contextPaths) {
+        return new ContextPathServicesBuilder<>(this, defaultVirtualHostBuilder, contextPaths);
```

Comment:
How about
- Changing the constructor parameter of `ContextPathServicesBuilder` to `Set<String>`
- Calling `contextPath(Iterable<String> contextPaths)` here?
```java
public ContextPathServicesBuilder<ServerBuilder> contextPath(String... contextPaths) {
    return contextPath(ImmutableSet.copyOf(requirNonNull(contextPaths, "contextPaths"));
}

public ContextPathServicesBuilder<ServerBuilder> contextPath(Iterable<String> contextPaths) {
    requirNonNull(contextPaths, "contextPaths")
    return new ContextPathServicesBuilder<>(
            this, defaultVirtualHostBuilder, ImmutableSet.copyOf(requirNonNull(contextPaths, "contextPaths"));
}
```
You might want to refer to this:
https://github.com/line/armeria/blob/13b0caa3a31421ca12f5821085dee751e4a54276/core/src/main/java/com/linecorp/armeria/client/AbstractDnsResolverBuilder.java#L306-L318

Reply:
Done! Thanks for the detailed review 😄